ros/src/tl_detector/tl_detector.py

```diff
@@ -283,14 +295,31 @@ def process_traffic_lights(self):
                  (specified in styx_msgs/TrafficLight)
 
         """
+
+        # searching_distance_tl parameter sets distance in which
+        # traffic lights will be searched as number of waypoints
+        searching_distance_tl = 120
```

Comment:
For better understanding, it would be good to specify the units of this distance. Right now it's not meters, but indices within the `waypoints` array.